docs/source/standard/contract.rst

```diff
@@ -94,3 +94,111 @@ Schema
 
     |ocdsDescription|
     All documents and attachments related to the contract, including any notices.
+
+.. _ContractTerms:
+
+ContractTerms
+=============
+
+Schema
+------
+
+:contractType:
+    string, required
+
+    The only choice currently is ``lease``.
+
+:leaseTerms:
+    :ref:`LeaseTerms`
+
+    The options of property lease.
+
+.. _LeaseTerms:
+
+LeaseTerms
+==========
+
+Schema
+------
+
+:leaseDuration:
+    ISO8601 duration object, required
+
+    Property lease duration in contract.
+
+:taxHolidays:
+    list of :ref:`TaxHolidays`
+
+    Special rated period of lease.
+
+:escalationClauses:
+    list of :ref:`EscalationClauses`
+
+.. _TaxHolidays:
+
+TaxHolidays
+===========
+
+Schema
+------
+
+:taxHolidaysDuration:
+    ISO8601 duration object, required
+
+    Duration of rent holidays.
+
+:conditions:
+    string, required
+
+    Lease rent holidays conditions in Ukrainian.
```

Comment:
The description of the specific conditions within the period during which the concessions are made for some reason.